UPGRADING.md

```diff
@@ -1,6 +1,37 @@
 Guzzle Upgrade Guide
 ====================
 
+6.0 to 7.0
+----------
+In order Guzzle to take advantage of the new features of PHP, we dropped the support of PHP 5.
+The minimum supported PHP version is now PHP 7.2.
+
+Wherever it was possible type hints and return types for functions and methods are applied. 
+
+Please make sure:
+- If you have enabled strict types that you are calling a function or a method with the correct type.
+- If you extend a class of Guzzle and you are overriding a method make sure you add the return type and type hints of the overriden method.
+
+Moreover:
+- All internal native functions calls of Guzzle are now prefixed with a slash. 
+This change makes it impossible for the users of the library to change the behavior of any of these functions
+
+Before:
+`curl_version()`
+
+After:
+`\curl_version()`
+ 
```

Comment:
```suggestion

- [PSR-18](https://www.php-fig.org/psr/psr-18/) support.
```

I think it's worth to mention this

Reply:
WDYT if on the first paragraph we had this: 
```
In order Guzzle to take advantage of the new features of PHP, and support [PSR-18](https://www.php-fig.org/psr/psr-18/) 
we dropped the support of PHP 5. The minimum supported PHP version is now PHP 7.2.
```

The reason is that Upgrade guide should have only the details that you need to update your version. The rest should be on the changelog as we currently have them...